js/tabs.js

```diff
@@ -7,14 +8,36 @@ function setupToolsTabs() {
     tabContents.removeClass('current');
   }
 
+  function selectTool(id) {
```

Comment:
Please add comments (what is the purpose of this function).

Reply:
Done! LMK if it's not clear enough.